components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -4229,6 +4302,23 @@ subroutine ocn_diagnostics_init(domain, err)!{{{
          end if
       end if
 
+      if (trim(config_thickness_drag_type) == 'centered') then
+         thickEdgeDragChoice = thickEdgeDragCenter
+         call mpas_log_write('Thickness drag option of ' //&
+              & trim(config_thickness_drag_type) // &
+              & 'is assigned')
+      elseif (trim(config_thickness_drag_type) == 'harmonic-mean') then
+         thickEdgeDragChoice = thickEdgeDragHarMean
+         call mpas_log_write('Thickness drag option of ' //&
+              & trim(config_thickness_drag_type) // &
+              & 'is assigned')
+      else
+         thickEdgeFluxChoice = thickEdgeFluxUnknown
+         call mpas_log_write('Thickness drag option of ' //&
+              & trim(config_thickness_drag_type) // &
+              & 'is not known', MPAS_LOG_CRIT)
```

Comment:
```suggestion
         call mpas_log_write('Thickness drag option of ' //&
              & trim(config_thickness_drag_type) // &
              & 'is not known', MPAS_LOG_CRIT)
```

Reply:
Also, this refers to "Flux" but should be "Drag" if it is kept.

Reply:
I would remove `thickEdgeFluxChoice = thickEdgeFluxUnknown` in the set of ifs above, too.